src/Splunk.Logging.Common/Splunk.Logging.Common.csproj

```diff
@@ -1,12 +1,13 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net45</TargetFrameworks>
+    <TargetFrameworks>net452;net6.0</TargetFrameworks>

```

Comment:
can we add support for net462

Reply:
Net452 is fine - net 462 project can happily reference the 452 build